config/initializers/chewy.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# configure chewy
+require "./config/chewy"
```

Comment:
> This requires a file that is in effect one line Chewy.request_strategy = :active_job. Could we just put that one line here and remove the separate config file?

We could totally move the chewy config into this file 👍 

> ...so maybe you can remove the yml file as well and just configure everything in one place.

I prefer we keep the `.yml` because its easy to scan for important details like secrets.